astropy/io/votable/tree.py

```diff
@@ -3087,6 +3094,59 @@ def _parse_fits(self, iterator, extnum, config):
 
         return array
 
+    def _parse_parquet(self, iterator, config):
+        """
+        Functionality to parse parquet files that are embedded
+        in VOTables.
+        """
+        # looks like votable already has a "Table" imported.
+        from astropy.table import Table as Table2
```

Comment:
@pllim  - Unrelated, but I wonder whether we could do a deprecation for the Table here in favour of some less ambiguous name, e.g. `TableElement` or similar? Having two `Table` class in astropy is not exactly that great.
(Note: we've just did something similar in pyvo and renamed its `Table` class)

Reply:
As long as there is buy-in from votable users (mostly Archive people)? Should probably get the deprecation in soon if you want it for astropy v6 or wait. Don't want to surprise them near feature freeze.

Reply:
I've pinned the IVOA channel, and then will also open an issue about it.

Reply:
#15358